Slim/Schema.pm

```diff
@@ -54,9 +54,12 @@ use Slim::Schema::Debug;
 use Slim::Schema::RemoteTrack;
 use Slim::Schema::RemotePlaylist;
 
+use constant SCAN_WORKS_FOR_MY_CLASSICAL_GENRES => 2;
+
 my $log = logger('database.info');
 
 my $prefs = preferences('server');
+my $scanWorks = $prefs->get('worksScan');
```

Comment:
We can't keep `$scanWorks` in a global variable without refreshing it, as in a server context (eg. Browse Music Folder) the pref could change. You could make this conditional (`... if main::SCANNER`).

Reply:
I don't think that code would ever run outside the scanner, but "always expect the unexpected" so done.